javascript/test/dist/customer.js

```diff
@@ -0,0 +1,8 @@
his,2,e)},proto.sensr_proto.ZoneConfig.prototype.getPbox=function(){return r.Message.getWrapperField(this,g.PolygonBox,3)},proto.sensr_proto.ZoneConfig.prototype.setPbox=function(e){return r.Message.setWrapperField(this,3,e)},proto.sensr_proto.ZoneConfig.prototype.clearPbox=function(){return this.setPbox(void 0)},proto.sensr_proto.ZoneConfig.prototype.hasPbox=function(){return null!=r.Message.getField(this,3)},proto.sensr_proto.ZoneConfig.prototype.getType=function(){return r.Message.getFieldWithDefault(this,4,0)},proto.sensr_proto.ZoneConfig.prototype.setType=function(e){return r.Message.setProto3EnumField(this,4,e)},proto.sensr_proto.StreamMessage.repeatedFields_=[1,2],r.Message.GENERATE_TO_OBJECT&&(proto.sensr_proto.StreamMessage.prototype.toObject=function(e){return proto.sensr_proto.StreamMessage.toObject(e,this)},proto.sensr_proto.StreamMessage.toObject=function(e,t){var o,n={objectsList:r.Message.toObjectList(t.getObjectsList(),g.Object.toObject,e),zonesList:r.Message.toObjectList(t.getZonesList(),proto.sensr_proto.ZoneConfig.toObject,e),health:(o=t.getHealth())&&proto.sensr_proto.SystemHealth.toObject(e,o)};return e&&(n.$jspbMessageInstance=t),n}),proto.sensr_proto.StreamMessage.deserializeBinary=function(e){var t=new r.BinaryReader(e),o=new proto.sensr_proto.StreamMessage;return proto.sensr_proto.StreamMessage.deserializeBinaryFromReader(o,t)},proto.sensr_proto.StreamMessage.deserializeBinaryFromReader=function(e,t){for(;t.nextField()&&!t.isEndGroup();){switch(t.getFieldNumber()){case 1:var o=new g.Object;t.readMessage(o,g.Object.deserializeBinaryFromReader),e.addObjects(o);break;case 2:o=new proto.sensr_proto.ZoneConfig;t.readMessage(o,proto.sensr_proto.ZoneConfig.deserializeBinaryFromReader),e.addZones(o);break;case 3:o=new proto.sensr_proto.SystemHealth;t.readMessage(o,proto.sensr_proto.SystemHealth.deserializeBinaryFromReader),e.setHealth(o);break;default:t.skipField()}}return e},proto.sensr_proto.StreamMessage.prototype.serializeBinary=function(){var e=new r.BinaryWriter;return proto.sensr_proto.StreamMessage.serializeBinaryToWriter(this,e),e.getResultBuffer()},proto.sensr_proto.StreamMessage.serializeBinaryToWriter=function(e,t){var o=void 0;(o=e.getObjectsList()).length>0&&t.writeRepeatedMessage(1,o,g.Object.serializeBinaryToWriter),(o=e.getZonesList()).length>0&&t.writeRepeatedMessage(2,o,proto.sensr_proto.ZoneConfig.serializeBinaryToWriter),null!=(o=e.getHealth())&&t.writeMessage(3,o,proto.sensr_proto.SystemHealth.serializeBinaryToWriter)},proto.sensr_proto.StreamMessage.prototype.getObjectsList=function(){return r.Message.getRepeatedWrapperField(this,g.Object,1)},proto.sensr_proto.StreamMessage.prototype.setObjectsList=function(e){return r.Message.setRepeatedWrapperField(this,1,e)},proto.sensr_proto.StreamMessage.prototype.addObjects=function(e,t){return r.Message.addToRepeatedWrapperField(this,1,e,proto.sensr_proto.Object,t)},proto.sensr_proto.StreamMessage.prototype.clearObjectsList=function(){return this.setObjectsList([])},
```

Comment:
What is this file? Seems to be autogenerated

Reply:
Will put it in git ignore.